project.clj

```diff
@@ -10,7 +10,7 @@
                  [org.clojure/tools.reader "1.3.2"]
                  [environ "1.2.0"]
                  [com.taoensso/timbre "4.10.0"]
-                 [io.replikativ/hitchhiker-tree "0.1.7"]
+                 [io.replikativ/hitchhiker-tree "0.1.7"] ;; TODO: Change to the version of the hh-tree that has the upsert op.
```

Comment:
This will require the definition of the deserialization bits for konserve as well: https://github.com/replikativ/hitchhiker-tree/blob/master/src/hitchhiker/tree/bootstrap/konserve.cljc#L142

Reply:
> Why do you need the upsert op in the hitchhiker-tree repo? We should be able to define it in Datahike and I think it makes sense, because it contains bits that are not relevant on the hitchhiker-tree level. We should definitely reference it from the hitchhiker-tree documentation though.

Right this is not needed. Using hh-tree's  'enqueue' instead now.